service/mapper/transitions_test.go

```diff
@@ -803,7 +841,7 @@ func TestTransitions_IndexChain(t *testing.T) {
 		assert.Error(t, err)
 	})
 
-	// FIXME: Add tests for Guarantees
+	// TODO: Add tests for Guarantees
```

Comment:
Is there an issue for this?

Reply:
Yes it's part of https://github.com/optakt/flow-dps/issues/333 :) I guess I forgot to link it though. Should be fixed before this PR is merged though hopefully.

Reply:
I added the link in the code :+1: